Http/Middleware/RenderBlock.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Modules\Block\Http\Middleware;
+
+use Illuminate\Http\Response;
+use Modules\Block\Repositories\BlockRepository;
+
+class RenderBlock
+{
+    /** @var BlockRepository */
+    private $blockRepository;
+
+    public function __construct(BlockRepository $blockRepository)
+    {
+        $this->blockRepository = $blockRepository;
+    }
+
+    public function handle($request, \Closure $next)
+    {
+        /** @var Response $response */
+        $response = $next($request);
+
+        // do not render galleries on backend
```

Comment:
I assume this is supposed to be more generic like `do not render shortcodes...` ?

Reply:
yes ... I reused some of the code from my gallery module (submitted a while ago to the official website), missed this comment